SI/Sem.03/Pract.03/Task05.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>;
+using namespace std;
+
+int main()
+{
+    int targetNumber;
+    int counter = 0;
+    int guessNumber;
+
+    cout << "Please enter whole number from 0 to 100: ";
+    cin >> targetNumber;
+
+    do
+    {
+        cout << "Enter your guess: ";
+
+        cin >> guessNumber;
+        counter++;
+
+        if (guessNumber == targetNumber)
+        {
+            cout << "You won with " << counter << " attempts";
+            break;
+        }
+    } while (true);
```

Comment:
We are looping until the numbers are equal.
Can we avoid/change the while (true) ?